impeller/entity/contents/content_context.h

```diff
@@ -279,11 +277,17 @@ struct ContentContextOptions {
   bool is_for_rrect_blur_clear = false;
 
   struct Hash {
-    constexpr std::size_t operator()(const ContentContextOptions& o) const {
-      return fml::HashCombine(
-          o.sample_count, o.blend_mode, o.stencil_compare, o.stencil_operation,
-          o.primitive_type, o.color_attachment_pixel_format,
-          o.has_stencil_attachment, o.wireframe, o.is_for_rrect_blur_clear);
+    constexpr uint64_t operator()(const ContentContextOptions& o) const {
+      return static_cast<uint64_t>(o.sample_count) << 56 |                   //
+             static_cast<uint64_t>(o.blend_mode) << 48 |                     //
+             static_cast<uint64_t>(o.stencil_compare) << 40 |                //
+             static_cast<uint64_t>(o.stencil_operation) << 32 |              //
+             static_cast<uint64_t>(o.primitive_type) << 24 |                 //
+             static_cast<uint64_t>(o.color_attachment_pixel_format) << 16 |  //
+             // bools
+             static_cast<uint64_t>(o.has_stencil_attachment) << 2 |
```

Comment:
This approach for the bools is correct.
```suggestion
             (o.has_stencil_attachment ? 1 : 0) << 2
```

Reply:
Done